k-distribution/include/kframework/builtin/domains.md

```diff
@@ -1055,6 +1059,14 @@ module INT-SYMBOLIC [symbolic]
   rule X modInt N => X requires 0 <=Int X andBool X <Int N [simplification]
   rule X   %Int N => X requires 0 <=Int X andBool X <Int N [simplification]
 
+  rule  1 ^Int _ =>  1                           [simplification]
+  rule -1 ^Int N =>  1 requires N %Int 2 ==Int 0 [simplification]
+  rule -1 ^Int N => -1 requires N %Int 2 ==Int 1 [simplification]
```

Comment:
I'm not sure if `N %Int 2` will return 1, for negative even numbers, e.g., -2, -4, ....   Some backends may return `-1`, depending on their semantics of modulo.  You may want to say that it is _not equal_ to 0, to be more general.

Reply:
Got it - will address. Thanks!